tests/integration/test_vulnerability_detector/test_scan_results/test_scan_nvd_feeds.py

```diff
@@ -25,23 +29,33 @@
 test_data_path = os.path.join(current_test_path, 'data')
 configurations_path = os.path.join(test_data_path, 'wazuh_nvd_configuration.yaml')
 vulnerabilities_data_path = os.path.join(test_data_path, CUSTOM_NVD_VULNERABILITIES_1)
+custom_cpe_helper_data_path = os.path.join(test_data_path, CUSTOM_CPE_HELPER)
+custom_msu_data_path = os.path.join(test_data_path, CUSTOM_MSU)
 
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 SCAN_TIMEOUT = 40
 
+with gzip.open(MSU_PATH, 'wb') as msu:
+    with open(custom_msu_data_path, 'rb') as custom_msu:
+        msu.write(custom_msu.read())
+
+copy(custom_cpe_helper_data_path, CPE_HELPER_PATH)
```

Comment:
I think it would be interesting to have a function that compress the file. This would be useful to re-use the code in a future.

Reply:
Done in c700279